tests/utils/test_UniversalDeployer.py

```diff
@@ -100,3 +100,30 @@ async def test_deployment(deployer_factory, unique):
             salt,                     # salt
         ]
     )
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize('unique', [TRUE, FALSE])
+async def test_declare_and_deploy(deployer_factory, unique):
```

Comment:
what's the subject of the test? this seems to be testing the signer rather than the UDC

Reply:
Yeah, the signer is mostly the subject. I considered adding a test_signer.py but proposed to sneak this in here because the UDC is contextually necessary and checks the flow of declare/deploy. No opposition here if having a test_signer.py is preferred

Reply:
i think i'd separate the test yes, especially because it's testing a test module. it's a completely separated thing and it should have independent testing.

Reply:
Good point, sounds good to me!